src/Api/V1/Client/Js/client.js

```diff
@@ -97,14 +97,14 @@ jsClient.parameters =
 	},
 
 	eoo: true
-}
+};
 
 jsClient.ajax =
 {
 	init: function()
 	{
 		// a call was initialized
-		$('.submit').live('click', jsClient.ajax.submitFormHandler);
+		$('.submit').click(jsClient.ajax.submitFormHandler);
```

Comment:
Wouldn't `.on('click', jsClient.ajax.submitFormHandler);` be preferable here?  
It has the same functionality`live` has as opposed to just `click`. I don't know the context of how this is used but I think it'd be wise to keep the functionality the same and just update the deprecated stuff.  

See http://api.jquery.com/live/ for more information